dsgrid/config/representative_period_time_dimension_config.py

```diff
@@ -128,13 +138,14 @@ def convert_dataframe(
                     expr = f"{func}(local_time) AS {col}"
                     if col == "day_of_week":
                         expr = f"mod(dayofweek(local_time)+7-2, 7) AS {col}"
+                    elif col == "is_weekday":
```

Comment:
This is the only real change related to mapping of the times.

Reply:
Did you check out the comment at the top of the func? I think you need this:
```
f"(mod(dayofweek(local_time)+7-2, 7) < 5) AS {col}"
```

# in spark.dayofweek: 1=Sunday, 7=Saturday
# dsgrid uses python standard library (same for pandas), which has day_of_week: 0=Monday, 6=Sunday
# the mapping is: python.dt.day_of_week = [(i+7-2)%7 for i in spark.dayofweek]

Reply:
This is totally my bad. I read through all of this and the Spark documentation. I intended to use this function (https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.functions.weekday.html), but I guess I copy/pasted the line above and forgot to swap the functions. We should use `weekday` everywhere because it lines up with Python.